src/ducks/knownAccounts.ts

```diff
@@ -0,0 +1,55 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+import {
+  ActionStatus,
+  KnownAccount,
+  KnownAccountsInitialState,
+} from "types/types.d";
+
+export const fetchMemoRequiredAccountsAction = createAsyncThunk<KnownAccount[]>(
+  "knownAccounts/fetchKnownAccountsAction",
+  async () => {
+    let result;
+    try {
+      const response = await fetch(
+        "https://api.stellar.expert/explorer/directory?tag[]=memo-required",
+      );
+
+      if (response.ok) {
+        const data = await response.json();
+        result = data?._embedded?.records;
+      }
+    } catch (error) {
+      // Do nothing
+    }
+
+    return result;
+  },
+);
+
+const initialState: KnownAccountsInitialState = {
+  memoRequired: undefined,
```

Comment:
We might add `unsafe` and `malicious` checks in the future, does this make it clear?

Reply:
Yup, that makes sense. lGTM